check50/_api.py

```diff
@@ -263,7 +269,7 @@ def stdout(self, output=None, str_output=None, regex=True, timeout=3):
                 result += self.process.after
             raise Mismatch(str_output, result.replace("\r\n", "\n"))
         except TIMEOUT:
-            raise Failure(_("did not find {}").format(_raw(str_output)))
+            raise Failure(_("did not find {} within {} seconds").format(_raw(str_output), timeout))
```

Comment:
@dmalan @brianyu28  Do we want to expose the timeout of each check to students directly? e.g. if students don't print anything for hello, they'll see `did not find "hello, world\n" within 3 seconds" ` rather than just `did not find "hello, world\n"`. Otherwise, this looks aok to me.

Reply:
I can see the use case for a more advanced crowd to be able to actually see the timeouts. However, for problems like mario/caesar, I believe it's more confusing to see "within ... seconds" than not. As it might mistakenly cause you to believe that your program is too slow, whereas in reality your program is probably waiting for input erroneously.

Instead I added a `show_timeout` optional param to `.stdout` that lets you opt-in for the `within ... seconds` message.